packages/portal/docs/style/BootstrapVue/Toast.md

```diff
@@ -0,0 +1,61 @@
+Customised BootstrapVue toast component.
+
+Docs including more features: https://bootstrap-vue.org/docs/components/toast
+
+Used by:
+- EntityUpdateModal
+- NewFeatureNotification
+- ItemPinButton
+- ItemPinModal
+- RecommendationButtons
+- SearchInterface
+- SetDeleteModal
+- SetPublicationRequestWidget
+- SetPublishButton
+
+```jsx
+  <div class="p-3 bg-secondary progress-bar-striped" style="min-height: 170px;">
+    <b-button class="mb-2" variant="primary" @click="$bvToast.show('example-toast')">
+      Show toast
+    </b-button>
+    <b-button class="mb-2" variant="primary" @click="$bvToast.show('example-toast-2')">
+      Show white toast with buttons
+    </b-button>
+
+    <b-toast 
+      id="example-toast"
+      static
+      noCloseButton
```

Comment:
The props on this should be in kebab-case.

Reply:
Done.